src/core/js/views/drawerView.js

```diff
@@ -176,7 +176,13 @@ class DrawerView extends Backbone.View {
 
       const easing = Adapt.config.get('_drawer')?._showEasing || 'easeOutQuart';
       direction[this.drawerDir] = 0;
-      this.$el.velocity(direction, this.drawerDuration, easing);
+      this.$el.velocity(direction, {
+        duration: this.drawerDuration,
+        easing,
+        complete: () => {
```

Comment:
Could you use 'drawer:opened' from line 152 instead of introducing a new event?

Reply:
yes, Im going to close this PR, I have just pushed to https://github.com/adaptlearning/adapt-contrib-glossary/pull/50 and tested it out with that trigger and it seems to work fine for me.